generate/generate.go

```diff
@@ -142,7 +142,14 @@ func New() Generator {
 					Type: "mount",
 				},
 			},
-			Devices: []rspec.Device{},
+			Devices: []rspec.Device{
+				{
+					Path:  "/dev/fuse",
```

Comment:
`New()` is supposed to generate a config that the runtime-tools maintainers feel is a good default (without knowing anything about the container itself).  I don't think that config should contain`/dev/fuse` unless the user explicitly asks for it.